dags/taar_monthly.py

```diff
@@ -0,0 +1,80 @@
+"""
+This configures a monthly DAG to delete TAAR opt-out profiles.
+"""
+from airflow import DAG
+from datetime import datetime, timedelta
+from airflow.models import Variable
+from airflow.operators.sensors import ExternalTaskSensor
+
+from operators.gcp_container_operator import GKEPodOperator  # noqa
+
+TAAR_BIGTABLE_INSTANCE_ID = Variable.get("taar_bigtable_instance_id")
+TAAR_ETL_STORAGE_BUCKET = Variable.get("taar_etl_storage_bucket")
+TAAR_PROFILE_PROJECT_ID = Variable.get("taar_gcp_project_id")
+TAAR_DATAFLOW_SUBNETWORK = Variable.get("taar_dataflow_subnetwork")
+
+# This uses a circleci built docker image from github.com/mozilla/taar_gcp_etl
+TAAR_ETL_CONTAINER_IMAGE = (
+    "gcr.io/moz-fx-data-airflow-prod-88e0/taar_gcp_etl:0.6.2"
+)
+
+# should be in sync with shredder DAG
+SCHEDULE_DAYS = 28
+DELETE_DAYS = 29
+
+
+default_args_monthly = {
+    "owner": "epavlov@mozilla.com",
+    "email": [
+        "anatal@mozilla.com",
+        "mlopatka@mozilla.com",
+        "hwoo@mozilla.com",
+        "epavlov@mozilla.com",
+        "telemetry-alerts@mozilla.com"
+    ],
+    "depends_on_past": False,
+    # should be in sync with shredder DAG
+    "start_date": datetime(2020, 4, 7),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=60),
+}
+
+taar_monthly = DAG(
+    "taar_weekly", default_args=default_args_monthly, schedule_interval=timedelta(days=SCHEDULE_DAYS)
+)
+
+wait_shredder = ExternalTaskSensor(
```

Comment:
i think shredder may move out of airflow in the future

Reply:
And I don't think there's any need to ensure the date ranges are in sync with shredder. The deletion ping tables are updated daily. As long as this DAG is running every 28 days, it doesn't matter where the boundaries are.

Reply:
Yes, it makes sense. Actually, I have an even better idea: I can just attach this job to taar_weekly right after BigTable is filled and delete opt-out profiles for the last 28 days. Then even if shredder wasn't launched yet and deleted last week profiles were repopulated they will be deleted on the next step. It will allow reducing the delay of deletion after a user opts out. It's just backfilling will be a bit slower. I could also exclude those profiles on BigTable update step, but it overcomplicates things.